drivers/i2s/Kconfig.stm32

```diff
@@ -0,0 +1,143 @@
+# Kconfig - STM32 I2S driver configuration options
+#
+# Copyright (c) 2018 STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig I2S_STM32
+	bool "STM32 MCU I2S controller driver"
+	depends on I2S && SOC_SERIES_STM32F4X
+	select DMA
+	default n
+	help
+	  Enable I2S support on the STM32 family of processors.
+	  (Tested on the STM32F4 series)
+
+if I2S_STM32
+
+config I2S_STM32_RX_BLOCK_COUNT
```

Comment:
Is that something that can be tuned?
If yes, can it be put in dts?

Reply:
The DTS should be only used to describe the h/w not the s/w configuration. So, I don't think it is correct to put those flags into Device Tree.

Are they necessary? Well, I'm not sure in fact. I think I can fix them to 4 in the .h. But why not leave the possibility to configure them?

Reply:
ok, then keep them that way.